src/components/input-time-picker/input-time-picker.e2e.ts

```diff
@@ -244,7 +244,7 @@ describe("calcite-input-time-picker", () => {
     date.setSeconds(59);
 
     const expectedValue = "13:59:59";
-    const expectedDisplayValue = localizeTimeString(expectedValue, lang);
+    const expectedDisplayValue = localizeTimeString(expectedValue, lang, "latn");
 
     inputTimePicker.setProperty("value", expectedValue);
 
```

Comment:
Might be good to also add at least 1 arab numberingSystem test here as well?

Reply:
Agreed.